.gitignore

```diff
@@ -41,3 +41,9 @@ dist
 **/.terraform
 
 tmp
+
```

Comment:
Should we add a GraphQL action as part of `lint_test_build` to double check that everything is ok with this `frontend` package ? Maybe just running the `build` step?

Reply:
Sure. It will have to be a separate step in the workflow - we can not include `packages/frontend` in the root `tsconfig.json` as a reference (at this moment at least).

```
➜ rafiki (sj-rafiki-admin-ui) ✗ pnpm build                                                                                        8.351s  (sj-rafiki-admin-ui|✚) 12:35

> @interledger/rafiki@1.0.0 build /Users/radu/Code/rafiki
> tsc --build

node_modules/.pnpm/@remix-run+node@1.14.0/node_modules/@remix-run/node/dist/upload/fileUploadHandler.d.ts:47:22 - error TS2420: Class 'NodeOnDiskFile' incorrectly implements interface 'File'.
  Property 'prototype' is missing in type 'NodeOnDiskFile' but required in type 'File'.

47 export declare class NodeOnDiskFile implements File {
                        ~~~~~~~~~~~~~~

  node_modules/.pnpm/typescript@4.9.5/node_modules/typescript/lib/lib.dom.d.ts:2572:5
    2572     prototype: Blob;
             ~~~~~~~~~
    'prototype' is declared here.


Found 1 error.
```